data_structures/singly_linked_list.py

```diff
@@ -1,18 +1,12 @@
 class Node:
-  
-  value = None
-  succ = None
-
```

Comment:
You don't need to declare instance fields in Python. This syntax is usually for class fields.